Cargo.toml

```diff
@@ -88,18 +83,15 @@ itertools = "0.12"
 libc = "0.2"
 libloading = "0.8.5"
 linkme = "0.3"
-log = "0.4"
```

Comment:
They are not in use since we have removed the snforge test collector. 
TBH I am only removing them now, because I've updated my IDE yesterday and apparently the new version supports showing unused deps 😆 
We don't have unused deps check in CI.